packages/ember-application/lib/system/application.js

```diff
@@ -316,46 +295,19 @@ var Application = Namespace.extend(DeferredMixin, {
     Create a container for the current application's registry.
 
     @private
-    @method buildContainer
+    @method buildInstance
     @return {Ember.Container} the configured container
   */
-  buildContainer: function() {
+  buildInstance: function() {
     var container = this.__container__ = this.__registry__.container();
```

Comment:
> Are you suggesting that `ApplicationInstance.create().__container__` should be used instead of `ApplicationInstance.create().container`?

@rwjblue I wasn't suggesting this, but I'd recommend doing so *IF* `ApplicationInstance` becomes public itself. Since that doesn't appear to be the case in this PR, I'd hold off until then.

> Given that it is not particularly onerous to continue supporting it, at least until 2.0, we thought it best to preserve it.

@tomdale Those are my thoughts as well. Perhaps tag it with the `2.0TODO` flag mentioned above?